conda_index/index/sqlitecache.py

```diff
@@ -85,21 +85,21 @@ def __get__(self, inst, objtype=None) -> Any:
 
 
 class CondaIndexCache:
-    upstream_stage = "fs"
-
     def __init__(
         self,
         channel_root: Path | str,
         subdir: str,
         *,
         fs: MinimalFS | None = None,
         channel_url: str | None = None,
+        upstream_stage: str = "fs",
```

Comment:
We need this but it's awkward to pass both the class and its parameters separately to `class ChannelIndex()`